bin/search_solr_tools

```diff
@@ -22,10 +64,20 @@ class SolrHarvestCLI < Thor
       begin
         harvest_class = get_harvester_class(target)
         harvester = harvest_class.new(options[:environment], die_on_failure)
+        ping_status = SearchSolrTools::Helpers::HarvestStatus.new
+        ping_status.ping_solr = false unless harvester.ping_solr
+        ping_status.ping_source = false unless harvester.ping_source
+        raise SearchSolrTools::Errors::HarvestError, ping_status unless ping_status.ok?
+
         harvester.harvest_and_delete
+      rescue SearchSolrTools::Errors::HarvestError => e
+        puts "THERE WERE HARVEST STATUS ERRORS:\n#{e.message}"
```

Comment:
Does this application have an error log defined? Writing to stdout is ok, but I'd expect this sort of information to be logged.

Reply:
I don't think it does.  I'll see if I can add one today, if not I'll add a story to do that.